nav2_msgs/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   "action/FollowPath.action"
   "action/NavigateToPose.action"
   "action/Spin.action"
+  "action/DoDummyPrimitive.action"
```

Comment:
I suggest not using "Do" as a prefix on the action name. I think that is implied and we don't want to use "Do" for all of the actions.

Reply:
Agree, will change.